src/components/style/forms/dateInput.css

```diff
@@ -0,0 +1,67 @@
+/**
+ * DateInput styles
+ *
+ * Mostly `react-day-picker` overrides.
+ * Some styling is * handled in the component via the
+ * "style modifiers" API of `DayPicker`.
+ */
+
+@import "react-day-picker/lib/style.css";
+
+:root {
+  --fdsDateInput-control-height: 26px;
+}
+
+.fdsDateInput {
+  position: relative;
+  margin: 0;
+  color: var(--font-color-primary);
+}
+
+.fdsDateInput-navArrows {
+  position: absolute;
+  height: var(--fdsDateInput-control-height);
+  top: var(--space-half);
+  right: var(--space-half);
+}
+
+.fdsDateInput .DayPicker-wrapper {
+  padding: 0;
+}
+
+.fdsDateInput .DayPicker-Caption {
+  padding-top: var(--space-half);
+  margin: -8px;
```

Comment:
Negative margins 😉Welcome to the dark side

Reply:
🙈